packages/ra-ui-materialui/src/input/RadioButtonGroupInput.stories.tsx

```diff
@@ -10,14 +10,42 @@ import { FormInspector } from './common.stories';
 
 export default { title: 'ra-ui-materialui/input/RadioButtonGroupInput' };
 
-const choices = [
-    { id: 'M', name: 'Male' },
-    { id: 'F', name: 'Female' },
-];
-
 export const Basic = () => (
     <Wrapper>
-        <RadioButtonGroupInput choices={choices} source="gender" />
+        <RadioButtonGroupInput
+            source="category"
+            choices={[
+                { id: 'tech', name: 'Tech' },
+                { id: 'lifestyle', name: 'Lifestyle' },
+                { id: 'people', name: 'People' },
+            ]}
+        />
+    </Wrapper>
+);
+
+export const Row = () => (
+    <Wrapper>
+        <RadioButtonGroupInput
+            source="category"
+            choices={[
+                { id: 'tech', name: 'Tech' },
+                { id: 'lifestyle', name: 'Lifestyle' },
+                { id: 'people', name: 'People' },
+            ]}
+            row={false}
+        />
+    </Wrapper>
+);
+
+export const DefaultValue = () => (
+    <Wrapper>
+        <RadioButtonGroupInput
+            choices={[
+                { id: 'M', name: 'Male' },
+                { id: 'F', name: 'Female' },
+            ]}
+            source="gender"
+        />
```

Comment:
Did you forget to set the default value? Or do I miss the point of this Story?

Reply:
No, it's just that I replaced the previous "Basic" story with the same code as in the doc, to be able to make screenshots. I kept the previous Basic story and renamed it to DefaultValue, but it's useless and brings confusion, so I'll remove it.

Reply:
Or better, add a DefaultValue